util/inc/leatherman/util/time.hpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file
+ * Declares utility functions for dealing with time.
+ */
+#pragma once
+
+#include <string>
+#include <time.h>
```

Comment:
`time.h` is deprecated in C++ (http://www.cplusplus.com/reference/clibrary/), we should use `ctime`, and prefix its contents with `std::`.